packages/plugin-table/src/TableRowDomRenderer.ts

```diff
@@ -0,0 +1,30 @@
+import { AbstractRenderer } from '../../core/src/AbstractRenderer';
+import { TableRowNode } from './TableRowNode';
+import { DomRenderingEngine } from '../../plugin-dom/src/DomRenderingEngine';
+
+export class TableRowDomRenderer extends AbstractRenderer<Node[]> {
+    static id = 'dom';
+    engine: DomRenderingEngine;
+    predicate = TableRowNode;
+
+    /**
+     * Render the TableRowNode along with its contents.
+     */
+    async render(row: TableRowNode): Promise<Node[]> {
+        const tableSection = document.createElement(row.header ? 'thead' : 'tbody');
+        const domRow = document.createElement('tr');
+        this.engine.renderAttributes(row.attributes, domRow);
+        this.engine.renderAttributes(
+            row.attributes['table-section-attributes'] as Record<string, string>,
+            tableSection,
+        );
+        const renderedChildren = await this.renderChildren(row);
+        for (const renderedChild of renderedChildren) {
+            for (const domChild of renderedChild) {
+                domRow.appendChild(domChild);
+            }
+        }
```

Comment:
I'm already tired of this `for/for` construction. Nothing related to your branch, though.